server/gateway.go

```diff
@@ -2573,6 +2573,10 @@ func (c *client) gatewayAllSubsReceiveStart(info *Info) {
 	if account == "" {
 		return
 	}
+
+	c.Noticef("Gateway %q: switching account %q to %s mode",
+		info.Gateway, account, InterestOnly.String())
```

Comment:
You need .String()?

Reply:
no, I don't. Will remove. Thanks!

Reply:
ok feel free to merge after that change. LGTM